api/src/main/java/org/openmrs/module/eptsreports/metadata/HivMetadata.java

```diff
@@ -114,6 +114,14 @@ public Program getHIVCareProgram() {
 	
 	private String PATIENT_HAS_DIED = "PATIENT HAS DIED";
 	
+	private String TRANSFERED_OUT_TO_ANOTHER_FACILITY = "TRANSFERRED OUT TO ANOTHER FACILITY";
```

Comment:
Do we need to have these strings kept outside of the get function? Is there a benefit for doing it this way?

Reply:
@nthfloor these are just constants (missing the `final` and `static` though), I think we can leave them as class members so we can use in other methods if needed.

Reply:
Are these constants being used anywhere else? They don't seem reusable to me - it is just the name of the workflow-state.

Reply:
I usually just leave constants as class members. How should I fix this, use the string directly or declare the constant inside the method?

Reply:
Well I don't consider names as constants unless it is something I plan on reusing elsewhere. So in this case I would use the string directly